Platform/CoffeelakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -732,6 +733,7 @@ DEBUG_CODE_END();
     PlatformFeaturesInit ();
     break;
   case PreMemoryInit:
+    break;
```

Comment:
why do you add "break" here?   I think the intention was to call GpioInit() for both PreMemoryInit and PostMemoryInit.

Reply:
True. Corrected.